### ISSUE TITLE
Missing setup_env.sh

### ISSUE THREAD

**Issue by Lorilay2** [dot4qu/ikon-reservation-notifier#1]
After starting with NPM: 
> API key does not start with "SG.".
> Need to source setup_env.sh to set env variables. Make sure server is started with start script not manually

source setup_env.sh seems to be missing from repo?

**Comment by second-string** [comment]
Yeah, that's the file that contains all of the environment variables that are private. Stuff like my ikon username and password to query reservation data, the sendgrid api token for my account, and the ssl certs for running https. I've included the contents of the file with the keys omitted.

If you want to run this locally yourself, the basic variables you'd have to fill our are
`DEPLOY_STAGE=DEV`
`IKON_USERNAME=your ikon email`
`IKON_PASSWORD=your ikon password`
and the node options one.

You'll have to manually remove the checks in the code that look for those env variables to make sure their defined.
You'll also have to manually comment out any of the calls to the sendgrid functions (there are only a couple).

The last thing and the most complicated is that you'll have to generate https self-signed certificates to run it locally. Or, depending on how comfortable you are in the code, you could switch the server to an http one. All you'd have to do is switch the `const https = require("https")` to `const http = require("http")` and initialize the server with http with no certificates. That would let you access the website with a regular http connection.

If you don't want to deal with that certificate nonsense, I originally ran this as a little CLI script. You can see the unused remnants of that in `cli.js`. It would need to be updated a fair amount to use code that was written after it was completed, but all the pieces are in there. You'd just have to make the right calls

```echo "Setting DEPLOY_STAGE to prod..."
export DEPLOY_STAGE=
echo "done."

echo "Setting username env var..."
export IKON_USERNAME=
echo "done."

echo "Setting pw env var..."
export IKON_PASSWORD=
echo "done."

echo "Setting loose http parse to accept Ikon CDN Incapsula's malformed cookies..."
export NODE_OPTIONS=--insecure-http-parser
echo "done."

echo "Setting sendgrid api key..."
export SG_IKON_RESERVATION_KEY=
echo "done."

echo "Setting prod path to ssl key..."
export PROD_SSL_KEY_PATH=
echo "done."

echo "Setting prod path to ssl public cert..."
export PROD_SSL_CERT_PATH=
echo "done."

echo "Setting prod path to ssl CA cert..."
export PROD_SSL_CA_CERT_PATH=
echo "done."
```

**Comment by second-string** [comment]
The reason I have this open source is more for people to re-use portions of the Ikon auth system, it took me a bit of time to reverse-engineer. I didn't plan on people trying to spin it up on their own. It's doable, but I don't have any documentation written.
